modules/patient-portal/src/main/frontend/src/patient-portal/QuestionnaireSet.jsx

```diff
@@ -663,19 +664,19 @@ function QuestionnaireSet(props) {
   let introMessage = intro.replaceAll(pattern, getVisitInformation(pieces?.[1]) || pieces?.[2] || "");
 
   let welcomeScreen = (isComplete && isSubmitted || questionnaireIds?.length == 0) ? [
-    <Typography variant="h4" key="welcome-greeting">{ greet(username) }</Typography>,
+    greet(username),
     appointmentAlert(),
     displayText("noSurveysMessage", Typography, {color: "textSecondary", variant: "subtitle1", key: "survey-info"}),
   ] : [
-    <Typography variant="h4" key="welcome-greeting">{ greet(username) }</Typography>,
+    greet(username),
     appointmentAlert(),
     (introMessage
-      ? <FormattedText key="intro-message" className="patient-portal-instructions">{introMessage}</FormattedText>
-      : displayText("surveyIntro", Typography, {key: "welcome-message", className: "patient-portal-instructions"})
+      ? <FormattedText key="intro-message">{introMessage}</FormattedText>
+      : displayText("surveyIntro", Typography, {key: "welcome-message"})
     ),
-    <List key="welcome-surveys">
+    <List key="welcome-surveys" sx={{p : 0}}>
     { (questionnaireIds || []).map((q, i) => (
-      <ListItem key={q+"Welcome"}>
+      <ListItem key={q+"Welcome"} sx={{pt : 0, pb: 0}}>
```

Comment:
`py` should work iirc.

```suggestion
      <ListItem key={q+"Welcome"} sx={{py : 0}}>
```

Reply:
While this styling is according to the initial mockup, it looks neater with disablePadding:

<img width="747" height="1328" alt="image" src="https://github.com/user-attachments/assets/aa073df7-73de-479e-9d4d-271f3d50626b" />

versus

<img width="722" height="1320" alt="image" src="https://github.com/user-attachments/assets/eb1c18b8-e5f9-48f2-aca7-ec6b2b58fe32" />